package.json

```diff
@@ -90,23 +91,13 @@
     "@types/ua-parser-js": "^0.7.35",
     "@types/uuid": "^8.3.4",
     "@types/wcag-contrast": "^3.0.0",
-    "@typescript-eslint/eslint-plugin": "^4",
-    "@typescript-eslint/parser": "^4",
+    "@uniswap/eslint-config": "git+ssh://git@github.com:uniswap/eslint-config.git#da8c311",
```

Comment:
This is not going to work anywhere but locally, waiting for the initial release.